packages/cactus-common/src/main/typescript/index.ts

```diff
@@ -1,2 +1,17 @@
 export * from "./public-api";
 export { IListenOptions, Servers } from "./servers";
+export const LogLevel = {
+  TRACE: "TRACE",
+  trace: "trace",
+  ERROR: "ERROR",
+  error: "error",
+  WARN: "WARN",
+  warn: "warn",
+  INFO: "INFO",
+  info: "info",
+  DEBUG: "DEBUG",
+  debug: "debug",
+  SILENT: "SILENT",
+  silent: "silent",
+} as const;
+export type LogLevelDesc = (typeof LogLevel)[keyof typeof LogLevel];
```

Comment:
@ruzell22 
1. Please do not write code directly in index.ts, it is meant to collect the public API surface exports and that will very noisy if you start writing code directly in it.
2. Could you make the LogLevelDesc type a match to what is defined in `packages/cactus-common/node_modules/loglevel/index.d.ts` ? There's nothing wrong by the looks of it with what you have here, but it is the safest if we use the exact same definition that the library uses internally to reduce the risk of type compatibility issues later on.

```typescript
   /**
     * Log levels
     */
    interface LogLevel {
        TRACE: 0;
        DEBUG: 1;
        INFO: 2;
        WARN: 3;
        ERROR: 4;
        SILENT: 5;
    }

    /**
     * Possible log level numbers.
     */
    type LogLevelNumbers = LogLevel[keyof LogLevel];

    /**
     * Possible log level descriptors, may be string, lower or upper case, or number.
     */
    type LogLevelDesc = LogLevelNumbers
        | 'trace'
        | 'debug'
        | 'info'
        | 'warn'
        | 'error'
        | 'silent'
        | keyof LogLevel;
```

Reply:
@ruzell22 It checks out, thank you.